scripts/completion-tests/completionTests.sh

```diff
@@ -620,4 +593,4 @@ unalias helm
 
 # This must be the last call.  It allows to exit with an exit code
 # that reflects the final status of all the tests.
-_completionTests_exit
+_completionTests_exit
```

Comment:
Could you please keep the end-of-file new-line in place? Thanks

Reply:
Ok

Reply:
Done

Reply:
Thx